doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -257,20 +264,21 @@ Some operations imply that only a partial representation of an entity will be pr
 
 * `id` and `type` are not allowed in update operations, as they are inmutable properties.
 
-* In the requests in which entity `type` is allowed, it may be omitted. When omitted, the default string value `none`
-  is used for the type.
```

Comment:
Is `none` considered by the implementations as a valid value for the type of an entity ? 

If no, the `none` value should be declared as being a *reserved* keyword and implementation should trigger an error when a client submits the creation/update of an entity with an explicit type to `none`.

If yes, we have two scenario:
* either we treat the `none` as if the type has no type (like if it was omitted).
* either the value `none` is considered as a type but when listing entities we cannot make the diffrence between entities with no type and entity of type `none` (surely not a good thing).

Could this be made more explicit in the specification ?